qiskit/providers/ibmq/job/ibmqjob.py

```diff
@@ -404,6 +405,10 @@ def job_id(self):
     def submit(self):
         """Submit job to IBM-Q.
 
+        Events:
+            Publishes job creation using `ibmq.job.start`
```

Comment:
Can you change this to be more in line with the other places where we document emitted events?

```
Events:
     name.of.the.event: Explanation for the event
```

Reply:
Ok, had no idea we actually had onther `Events` docs

Reply:
Not many indeed, but hopefully we can make use of the pub/sub mechanism in other places eventually - thanks for adjusting!